xapian-core/include/xapian/diversify.h

```diff
@@ -46,17 +46,20 @@ class XAPIAN_VISIBILITY_DEFAULT Diversify {
     /// Top-k documents of given mset are diversified
     unsigned int k;
 
-    /// Store given mset as a vector of points
-    std::vector<Point> points;
+    /// MPT parameters
+    double lambda, b, sigma_sqr;
 
-    /// Store relevance score for each document of given mset
-    std::unordered_map<Xapian::docid, double> weights;
+    /// Store each document from given mset as a point and its relevance score
```

Comment:
The comment here suggests the scores are stored in this member, i.e. `points` but the scores are actually in the member below - `scores`.

Reply:
I must've forgot to correct the location of the comment after some code refactoring. Fixed in a fixup commit.